vivisect/qt/funcgraph.py

```diff
@@ -322,7 +395,72 @@ def _histSelected(self, expr):
             pass
         self.enviNavGoto(expr)
 
+    def VWE_SYMHINT(self, vw, event, einfo):
+        va, idx, hint = einfo
+        self.mem_canvas.renderMemoryUpdate(va, 1)
+        if vw.getFunction(va) == self.fva and self._canHazAutoRefresh():
+            self.refresh()
+
+    def VWE_ADDLOCATION(self, vw, event, einfo):
+        va, size, ltype, tinfo = einfo
+        self.mem_canvas.renderMemoryUpdate(va, size)
+
+    def VWE_DELLOCATION(self, vw, event, einfo):
+        va, size, ltype, tinfo = einfo
+        self.mem_canvas.renderMemoryUpdate(va, size)
+
+    def VWE_ADDFUNCTION(self, vw, event, einfo):
+        va, meta = einfo
+        self.mem_canvas.renderMemoryUpdate(va, 1)
+
+    def VWE_SETFUNCMETA(self, vw, event, einfo):
+        fva, key, val = einfo
+        self._updateFunction(fva)
+
+    def VWE_SETFUNCARGS(self, vw, event, einfo):
+        fva, fargs = einfo
+        self._updateFunction(fva)
+
+    def VWE_COMMENT(self, vw, event, einfo):
+        va, cmnt = einfo
+        self.mem_canvas.renderMemoryUpdate(va, 1)
+        if vw.getFunction(va) == self.fva and self._canHazAutoRefresh():
+            self.refresh()
+
+    @idlethread
+    def VWE_SETNAME(self, vw, event, einfo):
+        va, name = einfo
+        self.mem_canvas.renderMemoryUpdate(va, 1)
+        for fromva, tova, rtype, rflag in self.vw.getXrefsTo(va):
+            self.mem_canvas.renderMemoryUpdate(fromva, 1)
+        if vw.getFunction(va) == self.fva and self._canHazAutoRefresh():
+            self.refresh()
+
+    @idlethread
+    def VTE_IAMLEADER(self, vw, event, einfo):
+        user, fname = einfo
+
+        def setFollow():
+            self._following = einfo
+            self.updateWindowTitle()
+
+        self._follow_menu.addAction('%s - %s' % (user, fname), setFollow)
+        # FIXME: manage this list in a central location for the VivGui.
+        # FIXME: add "DONTFOLLOWME" to stop (and remove entries)
+        # FIXME: change what's shared to uniquely identify each channel
```

Comment:
Some of these sound fairly important. Maybe we should just tackle them now?

Reply:
agreed.  this PR is slightly more than a draft, but less of a push.  it's green.  sorry, and i appreciate your feedback and critical thought.

Reply:
btw, i'm also considering adding a "change nickname" functionality to allow me to change my name to "rakuy0" instead of my login name.  especially for people with logins like "tim" where there may be many collisions.

Reply:
I'm cool with the change nickname options, though that belongs more in the main portion of vivisect's ui (or even, dare I say it, in vqt).